packages/fhir-converter/src/templates/cda/Sections/Medication.hbs

```diff
@@ -18,12 +18,19 @@
                                 {{/with}}
                                 {{>References/MedicationRequest/medicationReference.hbs ID=(generateUUID (toJsonString this.supply)) REF=(concat 'Medication/' (generateUUID (toJsonString medEntry.substanceAdministration.consumable)))}},
                             {{/if}}
+                            {!-- I am keeping what was originally here but I have yet to see an author placed here --}}
                             {{#if medReq.supply.author.assignedAuthor}}
                                 {{#with (evaluate 'Utils/GeneratePractitionerId.hbs' obj=medReq.supply.author.assignedAuthor) as |practitionerId|}}
                                     {{>Resources/Practitioner.hbs practitioner=medReq.supply.author.assignedAuthor ID=practitionerId.Id}},
                                     {{>References/MedicationRequest/requester.hbs ID=(generateUUID (toJsonString ../supply)) REF=(concat 'Practitioner/' practitionerId.Id)}}
                                 {{/with}}
+                            {{else if ../this.substanceAdministration.author.assignedAuthor}}
```

Comment:
MedicationRequests missing provider information based on cx QA. Based on looking through a lot of CDAs, the practitioner was located in a different place than were we were originally checking for it. I made it an if else incase the original` medReq.supply.author.assignedAuthor` location does actually occur but we just havent seen it